### PR TITLE
#61 Added Interactions to comment

### PR DESCRIPTION
**Pull request by tclaus**
This is the API representation of likes of comments.
Pr for #61

### CODE REVIEW

File: _routes/comments.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ GET /api/v1/posts/:post_guid/comments
     },
     "body": "What a wonderful post!",
     "mentioned_people": [],
+    "interactions": {
+      "likes": [],
+      "likes_count": 0
+    }
```
Comment (jhass) [inline _routes/comments.md]:
```suggestion
    },
```

**Review by jhass — APPROVED** [review]

**Comment by jhass** [comment]
On second thought, any reason why we would not add API routes to like and unlike comments at the same time?

**Comment by tclaus** [comment]
> On second thought, any reason why we would not add API routes to like and unlike comments at the same time?

Is this a rhetoric question or a request? 

What did you think to just release the desktop- version now, and implement API next in a new PR? You now: Cathedral and Bazaar?

**Comment by jhass** [comment]
I'm asking because this only adds documentation for comment responses, but no new routes to like or unlike a comment.

**Comment by jhass** [comment]
Now that likes on comments are merged, I updated this to the actual implementation.

**Review by SuperTux88 — APPROVED** [review]

**Pull request merged**